lib/engines/content_object_store/test/factories/content_block_edition.rb

```diff
@@ -3,6 +3,7 @@
     details { "{}" }
     created_at { Time.zone.now.utc }
     updated_at { Time.zone.now.utc }
+    content_block_document { nil }
```

Comment:
Yeah, good call, I defaulted it to `nil` because I didn't want to mess with the model tests (that confirm a document is created), but it's better to specify a `nil` document in those tests and have the factory create the correct thing.